packages/app-mobile/components/note-item.js

```diff
@@ -128,13 +129,20 @@ class NoteItemComponent extends Component {
 
 		const selectionWrapperStyle = isSelected ? this.styles().selectionWrapperSelected : this.styles().selectionWrapper;
 
+		const noteTitle = Note.displayTitle(note);
+
 		return (
 			<TouchableOpacity onPress={() => this.onPress()} onLongPress={() => this.onLongPress()} activeOpacity={0.5}>
 				<View style={selectionWrapperStyle}>
 					<View style={opacityStyle}>
 						<View style={listItemStyle}>
-							<Checkbox style={checkboxStyle} checked={checkboxChecked} onChange={checked => this.todoCheckbox_change(checked)} />
-							<Text style={listItemTextStyle}>{Note.displayTitle(note)}</Text>
+							<Checkbox
+								style={checkboxStyle}
+								checked={checkboxChecked}
+								onChange={checked => this.todoCheckbox_change(checked)}
+								accessibilityLabel={_('to-do: %s', noteTitle)}
```

Comment:
You shouldn't include the note title in the translation. It'll translate the entire string. If you do want the titles translated, translate to-do and noteTitle separately.

Reply:
It looks like only the format string is translated (and not the format args):

https://github.com/laurent22/joplin/blob/d293474402403f9f9af271b0947bebc571984295/packages/lib/locale.ts#L583-L586

Reply:
Right, i misread that. I thought you were performing the formatting yourself.

I shouldn't be reviewing code from my phone

Reply:
That's fine! Thank you for reviewing the code!